### PR TITLE
DEVDOCS-4668: [update] GraphQL overview, clarify intent of Revoke a token endpoint

### PR DESCRIPTION
**Pull request by bc-andreadao**
# [DEVDOCS-4668]

Added note about revoking only compromised tokens

This PR is related to [PR 1115](https://github.com/bigcommerce/api-specs/pull/1115) (api doc)

[DEVDOCS-4668]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4668?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by rafa-avila-bc — APPROVED** [review]

File: docs/api-docs/storefront/graphql/graphql-api-overview.md [hunk 1]
```diff
@@ -133,6 +133,8 @@ Content-Type: application/json
 }
 ```
 
+Only use the [Revoke a token](/api-reference/store-management/tokens/api-token/revoketoken) endpoint to revoke compromised tokens under emergency situations. Uncompromised short-lived tokens do not need to be revoked.
```
Comment (bookernath) [inline docs/api-docs/storefront/graphql/graphql-api-overview.md:136]:
```suggestion
Only use the [Revoke a token](/api-reference/store-management/tokens/api-token/revoketoken) endpoint to revoke compromised tokens under emergency situations. Uncompromised short-lived tokens do not need to be revoked and should be allowed to naturally expire.  

```

**Review by bookernath — APPROVED** [review]
LGTM, made suggestion for improved verbiage 🍹

File: docs/api-docs/storefront/graphql/graphql-api-overview.md [hunk 2]
```diff
@@ -133,6 +133,9 @@ Content-Type: application/json
 }
 ```
 
+Only use the [Revoke a token](/api-reference/store-management/tokens/api-token/revoketoken) endpoint to revoke compromised tokens under emergency situations. Uncompromised short-lived tokens do not need to be revoked and should be allowed to naturally expire.
```
Comment (bc-traciporter) [inline docs/api-docs/storefront/graphql/graphql-api-overview.md]:
```suggestion
Only use the [Revoke a token](/api-reference/store-management/tokens/api-token/revoketoken) endpoint to revoke compromised tokens under emergency situations. Uncompromised short-lived tokens do not need to be revoked and should be allowed to expire naturally.  

```

**Review by bc-traciporter — COMMENTED** [review]
I made a suggestion, you can take it or leave it as is.

**Review by stanzikratelbc — APPROVED** [review]
LGTM. Ready to merge

**Pull request merged**